taipy/common/config/_serializer/_base_serializer.py

```diff
@@ -105,9 +103,18 @@ def _from_dict(cls, as_dict) -> _Config:
                         sect_as_dict, None, None
                     )
                 elif issubclass(section_class, Section):
-                    config._sections[section_name] = cls._extract_node(as_dict, section_class, section_name, config)
+                    config._sections[section_name] = cls.__extract_node(as_dict, section_class, section_name, config)
         return config
 
+    @staticmethod
+    def __extract_node(config_as_dict, cls_config, node, config: Optional[Any]) -> Dict[str, Section]:
+        res = {}
+        for key, value in config_as_dict.get(node, {}).items():  # my_task, {input=[], output=[my_data_node], ...}
+            key = _validate_id(key)
+            res[key] = cls_config._from_dict(value, key, config)  # if config is None else cls_config._from_dict(key,
+            # value, config)
```

Comment:
I believe the comment can be removed safely now

```suggestion
            res[key] = cls_config._from_dict(value, key, config)
```